frontend/src/models/user.ts

```diff
@@ -1,3 +1,9 @@
 export interface IUser {
   email: string
+}
+
+export interface IUserPreference{
+  nickname: string
+  email: string
+  id: string
```

Comment:
is id a string?

Reply:
We dont have any id by now from backend, but I corrected all to string type!